src/components/dashboard/Dashboard.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import Skills from '~/components/skills/Skills';
 import Sidenav from '~/components/sidenav/Sidenav';
+import AddSkill from '~/components/form/skill/AddSkill';
+import AddProject from '~/components/form/project/AddProject';
 import Certificate from '~/components/certificate/Certificate';
 import Achievements from '~/components/achievements/Achievements';
 import WorkExperience from '~/components/workexperience/WorkExperience';
+import AddContactInformation from '~/components/form/contact/AddContact';
```

Comment:
wow, so many components are being imported. Will it all be used in this component.

Reply:
The form component can be removed. Will make the changes.

Reply:
Fixed.